frontend/components/top_nav/SiteTopNav/SiteTopNav.tests.tsx

```diff
@@ -0,0 +1,351 @@
+    );
+
+    await user.click(screen.getByTestId("user-menu"));
+
+    expect(screen.getByText(/hosts/i)).toBeInTheDocument();
+    expect(screen.getByText(/controls/i)).toBeInTheDocument();
+    expect(screen.getByText(/software/i)).toBeInTheDocument();
+    expect(screen.getByText(/queries/i)).toBeInTheDocument();
+    expect(screen.getByText(/schedule/i)).toBeInTheDocument();
+    expect(screen.getByText(/policies/i)).toBeInTheDocument();
+    expect(screen.getByText(/my account/i)).toBeInTheDocument();
+    expect(screen.getByText(/documentation/i)).toBeInTheDocument();
+    expect(screen.getByText(/sign out/i)).toBeInTheDocument();
+
+    expect(screen.queryByText(/settings/i)).not.toBeInTheDocument();
+    expect(screen.queryByText(/manage users/i)).not.toBeInTheDocument();
+  });
+  it("renders correct navigation for free global observer", async () => {
+    const render = createCustomRenderer({
+      context: {
+        app: {
+          isPremiumTier: false,
+        },
+      },
+    });
+
+    const { user } = render(
+      <SiteTopNav
+        config={createMockConfig()}
+        currentUser={createMockUser({
+          role: "observer",
+          global_role: "observer",
+        })}
+        location={urlLocation}
+        onLogoutUser={noop}
+        onNavItemClick={noop}
+      />
+    );
+
+    await user.click(screen.getByTestId("user-menu"));
+
+    expect(screen.getByText(/hosts/i)).toBeInTheDocument();
+    expect(screen.getByText(/software/i)).toBeInTheDocument();
+    expect(screen.getByText(/queries/i)).toBeInTheDocument();
+    expect(screen.getByText(/policies/i)).toBeInTheDocument();
+    expect(screen.getByText(/my account/i)).toBeInTheDocument();
+    expect(screen.getByText(/documentation/i)).toBeInTheDocument();
+    expect(screen.getByText(/sign out/i)).toBeInTheDocument();
+
+    expect(screen.queryByText(/controls/i)).not.toBeInTheDocument();
+    expect(screen.queryByText(/schedule/i)).not.toBeInTheDocument();
+    expect(screen.queryByText(/settings/i)).not.toBeInTheDocument();
+    expect(screen.queryByText(/manage users/i)).not.toBeInTheDocument();
+  });
+  it("renders correct navigation for premium global admin", async () => {
+    const render = createCustomRenderer({
+      context: {
+        app: {
+          isPremiumTier: true,
+          isGlobalAdmin: true,
+        },
+      },
+    });
+
+    const { user } = render(
+      <SiteTopNav
+        config={createMockConfig()}
+        currentUser={createMockUser()}
+        location={urlLocation}
+        onLogoutUser={noop}
+        onNavItemClick={noop}
+      />
+    );
+
+    await user.click(screen.getByTestId("user-menu"));
+
+    expect(screen.getByText(/hosts/i)).toBeInTheDocument();
+    expect(screen.getByText(/controls/i)).toBeInTheDocument();
+    expect(screen.getByText(/software/i)).toBeInTheDocument();
+    expect(screen.getByText(/queries/i)).toBeInTheDocument();
+    expect(screen.getByText(/schedule/i)).toBeInTheDocument();
```

Comment:
we'll want to either remove all `schedule` references here or as part of the [#7765](https://github.com/fleetdm/fleet/issues/7765) final merge to main